examples/ui/ui_scaling.rs

```diff
@@ -36,16 +36,19 @@ fn setup(mut commands: Commands, asset_server: ResMut<AssetServer>) {
 
     commands
```

Comment:
This example is broken for me in this PR:
![image](https://user-images.githubusercontent.com/2694663/214161960-07ef856b-f454-434d-84c3-5cc6b78c6b9e.png)